clang/lib/AST/Decl.cpp

```diff
@@ -3358,6 +3358,12 @@ bool FunctionDecl::isReservedGlobalPlacementOperator() const {
     return false;
 
   const auto *proto = getType()->castAs<FunctionProtoType>();
+  if (proto->getNumParams() < 2)
+    return false;
+  bool IsTypeAwareAllocator =
```

Comment:
is this bool used anywhere but in the 'return false' below?  If so, we probably don't need it.

Reply:
Yeah, I think this may date back to before a bunch of the helpers like isTypeIdentitySpecialization() so the logic was probably somewhat longer